packages/block-library/src/site-logo/edit.js

```diff
@@ -480,21 +482,25 @@ export default function LogoEdit( {
 
 		if ( ! media.id && media.url ) {
 			// This is a temporary blob image.
+			setTemporaryURL( media?.url );
 			setLogo( undefined );
 			return;
 		}
 
+		setTemporaryURL( media?.url );
 		setLogo( media.id, shouldForceSync );
 	};
 
 	const onRemoveLogo = () => {
+		setTemporaryURL();
```

Comment:
I think we can also remove the call here. The " Reset " control uses the' onRemoveLogo' callback, which is only displayed after the image upload.

Reply:
I tested it by removing this, It works fine.